File: CS/SchedulingDemo.Wpf/ViewModels/MonthViewDemoViewModel.cs
using System;
using System.Collections.Generic;
using System.Collections.ObjectModel;

namespace SchedulingDemo.ViewModels {
    public class MonthViewDemoViewModel {
        protected MonthViewDemoViewModel() {
            Start = DateTime.Today.AddDays(-14);
            Calendars = new ObservableCollection<TeamCalendar>(TeamData.Calendars);
            Appointments = new ObservableCollection<TeamAppointment>(TeamData.AllAppointments);
        }
        public virtual IEnumerable<TeamCalendar> Calendars { get; protected set; }
        public virtual IEnumerable<TeamAppointment> Appointments { get; protected set; }
        public virtual DateTime Start { get; set; }
    }
}

File: VB/SchedulingDemo.Wpf/ViewModels/MonthViewDemoViewModel.vb
Imports System
Imports System.Collections.Generic
Imports System.Collections.ObjectModel

Namespace SchedulingDemo.ViewModels
    Public Class MonthViewDemoViewModel
        Protected Sub New()
            Start = Date.Today.AddDays(-14)
            Calendars = New ObservableCollection(Of TeamCalendar)(TeamData.Calendars)
            Appointments = New ObservableCollection(Of TeamAppointment)(TeamData.AllAppointments)
        End Sub
        Public Overridable Property Calendars() As IEnumerable(Of TeamCalendar)
        Public Overridable Property Appointments() As IEnumerable(Of TeamAppointment)
        Public Overridable Property Start() As Date
    End Class
End Namespace
